framework/math/PETScUtils/petsc_forward_declarations.h

```diff
@@ -0,0 +1,13 @@
+#ifndef CHITECH_PETSC_FORWARD_DECLARATIONS_H
```

Comment:
This header file is a cheat code hack for forward declaring PETSc vectors and matrices.

Reply:
Why do we need this? (for my own edification)